### ISSUE TITLE
hedgenext and nginx

### ISSUE THREAD

**Issue by cbrown1** [libnewton/hedgenext-server#7]
I have my nextcloud server running behind nginx as a reverse proxy. How do I set up hedgenext to work?

**Comment by libnewton** [comment]
You can run the docker-compose file

**Comment by cbrown1** [comment]
I'm using the snap package, not docker. Does that change things?

**Comment by libnewton** [comment]
no, you can still deploy it via docker. Unfortunately, there are currently no instructions to setup HedgeNext. So you need some basic docker experience
